redash/models/__init__.py

```diff
@@ -772,6 +772,25 @@ def evaluate(self):
     def subscribers(self):
         return User.query.join(AlertSubscription).filter(AlertSubscription.alert == self)
 
+    def render_template(self):
+        data = json_loads(self.query_rel.latest_query_data.data)
+        context = {'rows': data['rows'], 'cols': data['columns']}
```

Comment:
How about we add to the context information about the alert like the state?

Reply:
I added `alert.state` and fixed help message.